src/common/playlists/PlayList.jsx

```diff
@@ -8,7 +10,11 @@ const PlayList = () => {
   if (plays.length === 0) {
     return (
         <div className="play-not-found">
-          <h2>Play Not Found</h2>
+          <ImageOops className="play-not-found-image" />
+          <p className='page-404-lead'>Play not found</p>
+          <p className='page-404-desc'>
+          Why don't you try anther play name or go back to <Link to="/" className='page-404-link'>home</Link>?
```

Comment:
Not only a wrong name can lead us to 404 page, but also an unsupported filter combination can be a reason to hit onto it. 
So the text needs to be updated accordingly.

Reply:
Its not a 404 page btw... 404 is page not found... it is play not found. There is a separate 404 page when a wrong route is hit. But I agree that we have change the sentence here.

Reply:
yeah.. not 404 page.. its actually '404 play' .. it just went in a flow 😊

Reply:
Here are some of my recommendations..

Option 1:
Just do a text fix without any link to any page.
I like this text... "Please change your search or adjust filters to find plays"

Option 2:
Another text fix but having a link to view all plays.
"Please change your search or adjust filters to find plays"
"View all plays" - leads to the playlist.

Reply:
@koustov please provide your suggestion so that we can close it.

Reply:
@nirmalkc let's go ahead with option 1 for now as koustov is out for an emergency now. Let us know when the changes are done.